### PR TITLE
Smallpatches

### PR DESCRIPTION
**Pull request by InfusOnWoW**

### CODE REVIEW

File: WeakAuras/Prototypes.lua [hunk 1]
```diff
@@ -7729,12 +7729,20 @@ Private.event_prototypes = {
...
         name = "movespeedpercent",
-        display = L["Movement Speed (%)"],
+        display = L["Current Movement Speed (%)"],
         type = "number",
         init = "GetUnitSpeed('player') / 7 * 100",
         store = true,
         conditionType = "number"
       },
+      {
+        name = "runpeedpercent",
```
Comment (nullKomplex) [inline WeakAuras/Prototypes.lua:7739]: Unfortunate typo.

**Review by nullKomplex — CHANGES_REQUESTED** [review]

File: WeakAuras/Prototypes.lua (hunk 1 above)
Comment (InfusOnWoW) [reply WeakAuras/Prototypes.lua]: Ah fixed, thanks.

File: WeakAuras/BuffTrigger2.lua [hunk 2]
```diff
@@ -967,6 +967,14 @@ local function TriggerInfoApplies(triggerInfo, unit)
     return false
   end
 
+  if triggerInfo.arenaSpec and unit:sub(1, 5) == "arena" then
+    -- GetArenaOpponentSpec doesn't use unit ids!
+    local i = tonumber(unit:sub(6))
+    if not triggerInfo.arenaSpec[GetArenaOpponentSpec(i)] then
```
Comment (mrbuds) [inline WeakAuras/BuffTrigger2.lua:973]: GetArenaOpponentSpec is nil on bcc & classic

Comment (InfusOnWoW) [reply WeakAuras/BuffTrigger2.lua:973]: That code path can never be encountered on classic/bcc

Comment (mrbuds) [reply WeakAuras/BuffTrigger2.lua:973]: Are you sure it can't be encounter if you import an aura with that option enable from Retail to BCC ?

Comment (InfusOnWoW) [reply WeakAuras/BuffTrigger2.lua:973]: Yes it checks if the unit id matches arena two lines above, and that unit id comes from the game.

**Pull request merged**